projects/1.16/assets/buddycards/patchouli_books/buddycard_guidebook/zh_cn/entries/buddysteel/perfect_buddysteel.json

```
@@ -0,0 +1,30 @@
+{
+  "name": "完美巴迪钢",
+  "icon": "buddycards:perfect_buddysteel_ingot",
+  "category": "buddycards:buddysteel",
+  "sortnum": 6,
+  "pages": [
+    {
+      "type": "text",
+      "text": "巴迪卡牌赋予了巴迪钢特殊的属性，因此通过使用紫砺、流光材料可以进一步增强巴迪钢，增强它与你收集卡牌的联系。这是非常值得的，因为一旦你完成卡牌收集，完美巴迪钢发挥的力量将远超巴迪钢。"
+    },
+    {
+      "type": "crafting",
+      "title": "完美巴迪钢粉",
+      "recipe": "buddycards:perfect_buddysteel_blend",
+      "text": "为了制作完美巴迪钢，你首先需要制作完美巴迪钢粉。这个过程需要一点巴迪钢，紫砺和流光，以及它们的块版本。"

```

In order to make perfect buddysteel, you must first make perfect buddysteel blend. This requires a bit of buddysteel, zylex, and luminis, both in a **massive** amount and a more concentrated item form.
又是 massive 又是 a bit of 的，不能理解。
```suggestion
      "text": "要制作完美巴迪钢，你首先得制作完美巴迪钢粉。这需要一点巴迪钢，紫砺和流光，再多一个它们的块。"
```